### DISCUSSION TITLE
亞錦賽，好想贏日本

### DISCUSSION THREAD

**Discussion by WeiYen1103** [1101-GS4719-C/Python-Community-GS4719-C#1761] [General]
今天晚上剛看完這屆的亞錦賽，對戰日本的冠軍場，我覺得大家真的都很棒了，真的都有盡力，昨天打菲律賓也守得很好！但感覺打擊可以再謹慎一些，選球打，但看著滿滿的人充斥著大巨蛋，有人拿著板子為台灣加油，有人揮舞者國旗，覺得很是熱血！！感覺又回到了之前的WBA，張育成轟出全壘打的時候，大家一起熱血沸騰的感覺，希望下次有機會可以去當場跟著大家一起為中華隊加油。
還有一幕也讓我應向蠻深刻的，前面日本投手投出了觸身球，打到打者的手腕，他也是很有禮貌的脫帽致歉，日本的一壘手也上前關心，真的是贏了球也贏了風度。

**Comment by uwje** [comment]
台灣隊投手的部分是發揮的很好的，但我認為打者的能力與抗壓性可能要再做提升，以台灣職棒的環境，感覺對於提升他們打擊能力感覺是沒幫助的，希望他們可以為了提升自己的能力，自費去其他國家或找特別的訓練師做額外訓練，畢竟我們的打者連對其他國家的業餘隊投手都打不好，所以要更佳正視自己打不好的原因，並去著重加強。

**Comment by BeaGle0928** [comment]
這次的台灣投手群確實相對給人一種比較安心的感覺，打擊上其實在前幾場的發揮也挺出色（雖然對手不在一個層級）。特別想說的是對上日本隊的兩場比賽，第一場延長賽突破僵局制，在戰術執行上有很大的問題，導致我們儘管在上半局讓日本隊只得一分，最終仍然輸球。冠軍戰的最後一個打席，陳聖平在滿球數的情況下打出了一隻界外全壘打，想必全場觀眾都覺得非常可惜，儘管最後以再見三振結束比賽，但他不放棄的奮鬥精神仍然非常值得讚許！期許未來選手們都能在自己的職業生涯中有很好的發展！
